Makefile

```diff
@@ -13,13 +10,16 @@ clippy:
 test:
 	cargo test --all-features --workspace
 
+test_sanitizer:
+	RUSTFLAGS="$(SANITIZER_FLAGS)" cargo test --all-features --workspace
+
 bench:
 	cargo bench --all-features --workspace
 
-ci: fmt clippy test run_release
+bench_sanitizer:
+	RUSTFLAGS="$(SANITIZER_FLAGS)" cargo bench --all-features --workspace
 
-proto: proto/proto/meta.proto proto/proto/rustproto.proto
```

Comment:
Why remove this?

Reply:
Now we use prost-build to generate `meta.rs` on cargo build, instead of generating them manually.